src/main/webapp/resources/js/pages/UserGroupsPage/components/UserGroupMembersTable.jsx

```diff
@@ -37,6 +38,13 @@ export default function UserGroupMembersTable({
   groupId,
   updateTable,
 }) {
+  const [total] = React.useState(members?.length);
+  const [paginationOptions, setPaginationOptions] = React.useState(null);
+
+  React.useMemo(() => {
```

Comment:
This should be:

```jsx
const paginationOptions = React.useMemo(() => {
  getPaginationOptions(total);
}, [total]);
```

Reply:
Updated in c96bd77d27718c9064e6535c7aac61dd11b3f80b